XamlAnimatedGif.Shared/XamlAnimatedGif.Shared.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;netstandard1.0</TargetFrameworks> <!-- Will add UWP TFM later -->
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Memory" Version="4.5.0" />
+    <PackageReference Include="System.Buffers" Version="4.5.0" />
+    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.6.40" PrivateAssets="All" />
```

Comment:
That's not how MSBuildSDKExtras is used :wink:
https://www.thomaslevesque.com/2018/11/23/making-a-wpf-app-using-a-sdk-style-project-with-msbuildsdkextras/

Reply:
Also, there's a more recent version

Reply:
I'm using an old'ish version of VS2017 (15.5.2) and the nuget SDK doesnt work on it :(

Reply:
Ah, yes, you need to update to 15.6 or later